.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "printWidth": 100,
-  "semi": false,
+  "printWidth": 80,
+  "semi": true,
   "singleQuote": true,
-  "bracketSpacing": false,
-  "jsxBracketSameLine": true,
```

Comment:
makes things more readable when set to false imo.

Reply:
hallelujah <3